build_fluka

```diff
@@ -22,5 +22,7 @@ fi
 
 # build source.f
 $FLUPRO/flutil/fff RadSource/source.f
+cp RadSource/source.o .
 # build the executable
-$FLUPRO/flutil/lfluka -L/$FLUPRO/ -l flukahp -L/$SRAGCODE_DIR/lib -a gcrsource -o flukahp RadSource/source.o
+#$FLUPRO/flutil/lfluka -m fluka -L/$FLUPRO/ -a flukahp -L/$SRAGCODE_DIR/lib -a gcrsource -a stdc++ -a hdf5 -a dl -o flukahp RadSource/source.o
+$FLUPRO/flutil/ldpmqmd -m fluka -L/$FLUPRO/ -a flukahp -L/$SRAGCODE_DIR/lib -a gcrsource -a stdc++ -L/data/opt/hdf5/lib -a hdf5 -a dl -o flukahp RadSource/source.o
```

Comment:
We need to use ldpm2qmd instead.  Looks like the ldpmqmd is the same as ldpm3qmd.  See FLUKA manual here http://www.fluka.org/fluka.php?id=man_onl&sub=102 with the little explanation, but instruction to use the older version of dpmjet for GCR simulation.   Below is the text from the manual.

DPMJET interaction model [Ran95,Roe01]. This model simulates the nucleus-nucleus collision above 5 GeV/nucleon. 
In case DPMJET is chosen for cosmic ray application, it is suggested to avoid the otherwise recommended DPMJET-III choice and to use instead the DPMJET-II.5 version (linking with the script $FLUPRO/flutil/ldpm2qmd).